client/packages/common/src/intl/locales/en/common.json

```diff
@@ -65,7 +65,7 @@
   "heading.additional-info": "Additional Info",
   "heading.are-you-sure": "Are you sure?",
   "heading.comment": "Comment",
-  "heading.custom-logo": "Custom logo",
+  "heading.custom-logo": "Custom logo (SVG)",
```

Comment:
If I can offer a suggestion..

<img width="590" alt="image" src="https://user-images.githubusercontent.com/9192912/233210301-85169685-dd20-413e-8e7f-5bb55c00ab52.png">

I think that it would benefit from having more detail.. so I would add in either an info icon ( as above ) or some inline text ( as below )

<img width="630" alt="image" src="https://user-images.githubusercontent.com/9192912/233211665-42100163-cd6a-4499-97a6-4d90d0508516.png">

[info icon version](https://github.com/openmsupply/open-msupply/compare/improve-custom-logo-description-suggestion)
[inline version](https://github.com/openmsupply/open-msupply/compare/improve-custom-logo-description-inline)

Reply:
I like the first option! will do when I get a minute